packages/x-components/src/design-system/components/suggestion-group/default.scss

```diff
@@ -27,6 +28,17 @@
   gap: var(--x-size-gap-suggestion-group-default);
   @include safari-gap(var(--x-size-gap-suggestion-group-default));
 
+  // border
+  border: none;
```

Comment:
border is a shorthand, this means that is overriding the border-color, border-width and border-style.

With this border: none you are disabling the rules before this for the border. If you want to disable de border by default, set the default tokes for border-width to 0.

And here set the boder-style to solid.